packages/webpack-loader/project.json

```diff
@@ -16,8 +16,7 @@
       "options": {
         "jestConfig": "packages/webpack-loader/jest.config.js",
         "passWithNoTests": true
-      },
-      "dependsOn": [{ "target": "build", "projects": "dependencies" }]
```

Comment:
I changed `@griffel/react` to use externals, so we need to build less code during each test and we don't need to build packages before